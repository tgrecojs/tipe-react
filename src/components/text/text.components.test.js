import React from 'react'
import { H1, H2, H3, H4, H5, H6 } from './index';
import renderer from 'react-test-renderer';

test(`<H1 />`, () => {
  expect(
    renderer.create(<H1 />).toJSON()
  ).toMatchSnapshot()
})