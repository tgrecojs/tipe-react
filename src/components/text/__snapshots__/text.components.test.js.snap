// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<H1 /> 1`] = `
<h1
  className="css-19fyqyq"
/>
`;
